tests/installation/partitioning/warning/zipl_small.pm

```diff
@@ -0,0 +1,47 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
```

Comment:
could you please update copyright for all files needed?

Reply:
Done.